app/grandchallenge/core/static/js/markdownx_full_page.js

```diff
@@ -0,0 +1,18 @@
+// Sync the height of the preview element to the height of the editor element.
+// This lets the user adjust the height of both elements by resizing the editor.
+$(document).ready(() => {
+    const ELEMENTS = document.getElementsByClassName("markdownx");
+    Object.values(ELEMENTS).map(element => {
```

Comment:
`getElementsByClassName` returns an HTMLCollection so `Object.values()` is incorrectly used here. Also `.map()` is used to create a new array so is also incorrectly used here. You can simply iterate over the elements using:

```
for (const element of elements) {
```

Reply:
Thanks for this! I have a tendency to avoid for-loops because they are slow in Python. I guess I didn't fully realize I'm writing in another language here :sweat_smile: